example/package.json

```diff
@@ -10,9 +10,11 @@
     "build:ios": "cd ios && xcodebuild -workspace IoReactNativeProximityExample.xcworkspace -scheme IoReactNativeProximityExample -configuration Debug -sdk iphonesimulator CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ GCC_OPTIMIZATION_LEVEL=0 GCC_PRECOMPILE_PREFIX_HEADER=YES ASSETCATALOG_COMPILER_OPTIMIZATION=time DEBUG_INFORMATION_FORMAT=dwarf COMPILER_INDEX_STORE_ENABLE=NO"
   },
   "dependencies": {
+    "@pagopa/io-react-native-nodelibs": "^0.0.2",
     "react": "18.2.0",
     "react-native": "0.72.6",
     "react-native-ble-manager": "^10.1.5",
+    "react-native-randombytes": "^3.6.1",
```

Comment:
`react-native-random-bytes` is marked as deprecated. What do you think to use `https://www.npmjs.com/package/react-native-get-random-values` as already used in `io-app`?

Reply:
I removed the dependency by introducing the new version of `react-native-nodelibs`